operator/src/network/controller.rs

```diff
@@ -157,6 +157,8 @@ pub const CERAMIC_LOCAL_NETWORK_TYPE: &str = "local";
 
 pub const BOOTSTRAP_JOB_NAME: &str = "bootstrap";
 
+const MAX_CERAMICS: usize = 10;
```

Comment:
Why is there a max number of ceramics?

Reply:
Keramik needs to clean up after itself if existing ceramics are removed. (i.e. network is configured with both a rust and go set, when the go set is removed it needs to be deleted). The max 10 is a simple way to do that, either its defined or its not, the ones that are not defined are deleted. k8s is inherently racy so simply storing in the status which ceramics had been previously defined doesn't provide a clean path to deleting old ones. 

And having 10 different configurations should be plenty for current use cases.

A better long term solution would be to use labels and query all resources with the matching labels and then delete the extra ones, but having a limit was a simple enough for now. However I could be convinced to do the labels logic now if we think that's better.